content/docs/14-troubleshooting/30-pcg.md

```diff
@@ -121,4 +124,135 @@ While these are the most common errors, some other issues might be related to th
 
 
 4. If the problem persists, contact our support team by sending an email to [support@spectrocloud.com](mailto:support@spectrocloud.com).
-<br />
\ No newline at end of file
+<br />
+
+
+# Scenario - IP Address Not Assigned During PCG Creation
+
+When deploying a PCG in VMware vCenter, the IP address fails to get assigned to the new node. This can be caused by networking errors such as DHCP issues or improper execution of cloud-init.
+<br />
+
+## Debug Steps
+<br />
+
+1. Verify the network connectivity between the new node and the DHCP server. Ensure that the DHCP server is operational and has available IP addresses in its pool.
+
+
+2. Check the network configuration on the new node and ensure that it is set to obtain an IP address automatically via DHCP.
+
+
+3. Review the cloud-init logs on the new node for any errors or warnings related to the IP address assignment process.
+<br />
+
+# Scenario - Cluster Provisioning Failure: "No Route to Host" to Kube API Server
+
+While provisioning a cluster, the event shows `no route to host` to the Kube API Server. This issue can occur due to improper networking configuration, errors in the cloud-init logs on the newly created VM, or sys logs indicating errors in the newly created VM.
```

Comment:
```suggestion
While provisioning a cluster, you may encounter an event log message stating `no route to host`.  This indicates an issue when the PCG cluster nodes are attempting to connect to the Kube API Server. This issue can occur due to improper networking configuration, or errors in the cloud-init process in the newly created VM. Use the following steps to resolve the route error.
```

This paragraph was difficult to read and needed touches to make it clearer to the reader what the scenario us.